sdk/monitor/Azure.Monitor.Query/src/autorest.md

```diff
@@ -6,9 +6,9 @@ Run `dotnet build /t:GenerateCode` to generate code.
 title: Query
 input-file:
     - https://github.com/Azure/azure-sdk-for-java/blob/1d14101ba93c6e616899c2ded93fbecb54699f84/sdk/monitor/azure-monitor-query/swagger/log_query_swagger.json
```

Comment:
This has no generated code changes, just re-aligning on the same set of swaggers with the team.